Formula/infrakit.rb

```diff
@@ -6,6 +6,8 @@ class Infrakit < Formula
       revision: "3d2670e484176ce474d4b3d171994ceea7054c02"
   license "Apache-2.0"
 
+  deprecate! because: "the source repo is unmaintained and archived"
```

Comment:
```suggestion
  deprecate! because: :repo_archived
```

It seems like most deprecations in homebrew-core use one of the [built-in reasons](https://github.com/Homebrew/brew/blob/c4ae961efac4904aabdd7dc3cbe20dc4ed004869/Library/Homebrew/deprecate_disable.rb#L10) if applicable, so it may be good to follow suit and use `:repo_archived` here.